### DISCUSSION TITLE
[814370] Monster Sanctuary  (本体)

### DISCUSSION THREAD

**Discussion by lifff55** [pjy612/SteamManifestCache#397] [Share]
`appid:814370`
keys:
```
814371;5e864a4651a6b57cd4b83c9fa76769c47d856d4adb1a4d8dc12ca79436589705
1475990;c15751148bcdb9c1ad11c3b1288edc93eea1d6979e7599f0f3e34906e1c2360c
```
code:862E467C984D29D3
email:1485943117@qq.com
[清单.zip](https://github.com/pjy612/SteamManifestCache/files/13989024/default.zip)

**Comment by ZEERDEER** [comment]
* 请附上清单

**Reply by lifff55** [reply]
怎么弄清单，
有教程吗，我是小白

**Reply by lifff55** [reply]
看见教程了，上传了

**Reply by pjy612** [reply]
查收

**Reply by lifff55** [reply]
大佬，我用新软件又新生成了一个code，之前用的旧版的13.5，新的code还能授权吗，(newcode:72F2C17CF79B5866)

**Reply by pjy612** [reply]
请看更新说明
